### DISCUSSION TITLE
add customAttribute tourAnchor in rectangle element

### DISCUSSION THREAD

**Discussion by adarshjalaja** [clientIO/joint#2475] [Q&A]
### Introduction

when I try to add a custom attribute or prop to rectangle element for touring guide (using nix ui tour) the property is splitter with hyphen like tour-anchor or touranchor we need a camel case attribute to be attached to it like tourAnchor (where A Is caps) 

tried adding this actionElement.prop("attrs/body/tourAnchor", "startWF");

and  its coming in html like tour-anchor="startWF" not tourAnchor="startWF"

is it possible to add same case attribute (tourAnchor) - camel case?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment]
An unofficial way (and the only one) to do that in `v3.7`:

```js
joint.dia.attributes.tourAnchor = { set: 'tourAnchor' };
el.attr('body/tourAnchor', true);
```
